File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1074.py
n = int(input())

for i in range(0,n):
    x = int(input())
    res = ""
    if(x==0):
        print("NULL")
    else:
        if(x%2==0):
            res+="EVEN "
        else:
            res+="ODD "
        if(x>0):
            res+="POSITIVE"
        else:
            res+="NEGATIVE"
        print(res)

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1074.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

n = parseInt(lines.shift())

for(i=0;i<n;i++){
    x = parseInt(lines.shift())
    res = ""
    if(x==0){
        console.log("NULL");
    } else{
        if(x%2==0){
            res+="EVEN "
        } else{
            res+="ODD "
        }
        if(x>0){
            res+="POSITIVE"
        } else{
            res+="NEGATIVE"
        }
        console.log(res)
    }
}